keyboards/handwired/sick_pad/kb.h

```diff
@@ -0,0 +1,20 @@
+#ifndef KB_H
+#define KB_H
```

Comment:
```suggestion
#pragma once
```

Prefer `#pragma once` to these old style include guards.  Also, please rename this file to "sick_pad.h".